msal/application.py

```diff
@@ -305,26 +305,71 @@ def acquire_token_silent(
                     "token_type": "Bearer",
                     "expires_in": int(expires_in),  # OAuth2 specs defines it as int
                     }
+        return self._acquire_token_silent_by_finding_rt_belongs_to_me_or_my_family(
+                the_authority, decorate_scope(scopes, self.client_id), account,
+                **kwargs)
 
+    def _acquire_token_silent_by_finding_rt_belongs_to_me_or_my_family(
+            self, authority, scopes, account, **kwargs):
+        query = {
```

Comment:
Should we have to add authority migration to look for aliases in this query as well ?

Reply:
In our current plan, the authority aliases would work at one layer higher, i.e. in the `acquire_token_silent(...)` level.

We will see how it works and continue this conversation at your PR there. Meanwhile, thanks for your review. I'm going to merge this now.